### PR TITLE
proxy server setup

### PR DESCRIPTION
**Pull request by yerx**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const app = express();
+
+app.use(cors());
+app.use(express.static('public'));
+
+app.engine('html', ejs.renderFile);
+app.get('/:id', (req, res) => {
+  res.render('../public/index.html');
+});
+app.set('view engine', 'html');
```
Comment (ga-possum) [inline server/index.js]: Dan and I figured out that we eventually don't need this line of code. (each of us somehow found it by himself)

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+app.use(proxy('/api/experiences', {target: 'http://localhost:3001/' }));
```
Comment (ga-possum) [inline server/index.js]: And seems that we did not use these 'proxy' lib also... But this is not to say that there is a problem, just another solution.

File: server/index.js (hunk 1 above)
Comment (yerx) [reply server/index.js]: Sounds good. I will test the proxy server without the line.

File: server/index.js (hunk 2 above)
Comment (yerx) [reply server/index.js]: For some reason, my proxy server did not work without the proxy library so I kept it in. Thanks.

**Pull request merged**